src/Components/Common/instanceHelpers.js

```diff
@@ -0,0 +1,21 @@
+import { AWS_PROVIDER, AZURE_PROVIDER } from './constants';
+
+export const instanceLink = (instanceID, provider, region) => {
+  switch (provider) {
+    case AWS_PROVIDER:
+      return `https://console.aws.amazon.com/ec2/home?region=${region}#InstanceDetails:instanceId=${instanceID}`;
+    case AZURE_PROVIDER:
+      return `https://portal.azure.com/#@rhdevcloudops.onmicrosoft.com/resource${instanceID}/overview`;
```

Comment:
It reads `resourceXXXX` immediately without any separator, is that correct?

Reply:
Also the tenant is hardcoded?

Reply:
Yeah, it is the Azure instanceID is of a format (I'm simplifying - but it starts with `/`) `/subscription/<sub_id>/resource_group/<resource_group_name>/Microsoft.Compute/<instance_name>`

Reply:
Gotcha and our tenant is hardcoded because that is the service account I guess.